src/components/featured-article/macro.spec.js

```diff
@@ -0,0 +1,133 @@
+                expect($('.ons-image__img').attr('alt')).toBe('Example alt text');
+            });
+        });
+    });
+
+    describe('GIVEN: Param: metadata', () => {
+        describe('WHEN: metadata with date and object is provided', () => {
+            const $ = cheerio.load(renderComponent('featured-article', EXAMPLE_FEATURED_ARTICLE_WITH_CHART));
+
+            test('THEN: it renders ISO date value correctly', () => {
+                expect($('time').attr('datetime')).toBe('2024-05-01');
+            });
+
+            test('THEN: it renders short date label correctly', () => {
+                expect($('time').text()).toBe('1 May 2024');
+            });
+
+            test('THEN: it displays metadata text', () => {
+                expect($('.ons-featured__item-attribute span').text()).toContain('Bulletin');
+            });
+        });
+
+        describe('WHEN: metadata object has no URL', () => {
+            const $ = cheerio.load(renderComponent('featured-article', EXAMPLE_FEATURED_ARTICLE_WITH_IMAGE));
+
+            test('THEN: it does not render metadata link', () => {
+                expect($('.ons-featured__attribute-link').length).toBe(0);
+            });
+
+            test('THEN: it still displays metadata text', () => {
+                expect($('.ons-featured__item-attribute span').text()).toContain('Article');
+            });
+        });
+
+        describe('WHEN: metadata is not provided', () => {
+            const articleWithoutMetadata = {
+                ...EXAMPLE_FEATURED_ARTICLE_WITH_CHART,
+                metadata: undefined,
+            };
+            const $ = cheerio.load(renderComponent('featured-article', articleWithoutMetadata));
+
+            test('THEN: metadata section is not rendered', () => {
+                expect($('.ons-featured__item-metadata').length).toBe(0);
+            });
+        });
+    });
+
+    describe('GIVEN: Params: description', () => {
+        describe('WHEN: description is provided', () => {
+            test('THEN: has expected description', () => {
+                const $ = cheerio.load(
+                    renderComponent('featured-article', { ...EXAMPLE_FEATURED_ARTICLE_WITH_CHART, description: 'Some description' }),
+                );
+                const title = $('.ons-featured__description').html().trim();
+                expect(title).toBe('Some description');
+            });
+        });
+    });
+
+    describe('GIVEN: Param: headingLevel', () => {
+        describe('WHEN: heading level is provided', () => {
+            const $ = cheerio.load(
+                renderComponent('featured-article', {
+                    ...EXAMPLE_FEATURED_ARTICLE_WITH_CHART,
+                    headingLevel: 3,
+                }),
+            );
+
+            test('THEN: title is rendered using correct heading tag', () => {
+                expect($('.ons-featured__title')[0].tagName).toBe('h3');
```

Comment:
```suggestion
                expect($('.ons-featured__title').tagName).toBe('h3');
```
Is this "[0]" needed? There should only be one anyway